vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Mathematical methods for `baselinenowcast`"
+description: "Description of mathematical methods used in the package"
+author: Kaitlyn Johnson
```

Comment:
I personally (and its epinowcast practice) dont like putting different authorship on vignettes vs the package but it happens (i.e I think we broke this in primarycensored but should really take it out if so).  Its no big deal either way though

Reply:
Just removed author